packages/README.md

```diff
@@ -2,7 +2,72 @@
 
 NB all Development Dependencies are in the top level package.json, none are in the packages.
 
-## [Components](./components)
+## Documentation index
+Please familiarise yourself with our:
+- [Code of conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md)
+- [Code Standards and Ways of Working](https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Workinnng.md)
```

Comment:
Typo. `https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Working.md`

Reply:
@mykeels done